etc/adminhtml/system.xml

```diff
@@ -90,15 +90,15 @@
                 <label>Payment method</label>
                 <field id="active" translate="label" type="select" sortOrder="10" showInDefault="1" showInWebsite="1"
                        showInStore="1">
-                    <label>Enable Two payments</label>
-                    <comment>Activates Two as a payment option in the checkout</comment>
+                    <label>Enable payment method</label>
+                    <comment>Activates this plugin as a payment option in the checkout flow</comment>
```

Comment:
do we want to describe this as a checkout flow or checkout page?

Reply:
You're right, checkout page is probably clearer. Done.